lib/socketclusterclient/parser.rb

```diff
@@ -0,0 +1,24 @@
+#
```

Comment:
I think we can use ruby enumerable here. Since enums were not present till python 3.x, we used numbers instead ...
One of the example is this one **https://github.com/sacOO7/socketcluster-client-java/blob/master/src/main/java/io/github/sac/Parser.java**

Reply:
Done.